arrow-data-source/standard/src/main/scala/com/intel/oap/spark/sql/execution/datasources/v2/arrow/ArrowScan.scala

```diff
@@ -63,4 +74,146 @@ case class ArrowScan(
   override def withFilters(partitionFilters: Seq[Expression],
                            dataFilters: Seq[Expression]): FileScan =
     this.copy(partitionFilters = partitionFilters, dataFilters = dataFilters)
+
+  // compute maxSplitBytes
+//  def maxSplitBytes(sparkSession: SparkSession,
+//                    selectedPartitions: Seq[PartitionDirectory]): Long = {
+//    // TODO: unify it with PREFERRED_PARTITION_SIZE_UPPER_BOUND.
+//    val defaultMaxSplitBytes = sparkSession.sessionState.conf.filesMaxPartitionBytes
+//    val openCostInBytes = sparkSession.sessionState.conf.filesOpenCostInBytes
+//    // val minPartitionNum = sparkSession.sessionState.conf.filesMinPartitionNum
+//    //  .getOrElse(sparkSession.leafNodeDefaultParallelism)
+//    val minPartitionNum = sparkSession.sessionState.conf.filesMinPartitionNum
```

Comment:
@jackylee-ch, I note you have introduced a sort of computation for taskParallelismNum, is it as same as minPartitionNum? This piece of code is ported from spark source code.

Reply:
No, they are not same. The `taskParallelismNum` is actually the `spark.sql.files.expectedPartitionNum`, which can be configured by the user and the default value is the maximum number of tasks that can be parallelized in the current application.